src/main.js

```diff
@@ -167,6 +178,95 @@ function onPageChange() {
       subtree: true,
     });
     console.info("Monitoring", [feed]);
+    return;
+  }
+
+  // FB5 design
+  // there's a feed div that we don't monitor yet
+  feed = document.querySelector("div[role=feed]:not([data-adblock-monitored])");
+  if (feed !== null) {
+    setFeedObserver();
+    return;
+  }
+  // there's a feed loading placeholder
+  feed = document.querySelector("div#suspended-feed");
```

Comment:
Can we use `getElementById()` here? Will that improve performance?

Reply:
sure.

Reply:
fixed